ion-schema/src/internal_traits.rs

```diff
@@ -79,13 +79,18 @@ pub(crate) trait WriteAsIsl<V: IslVersion>: Debug {
 /// In the future, it may include things such as:
 ///  - options to allow lossy conversion between ISL versions
 ///  - stylistic choices for writing as Ion
+#[derive(Copy, Clone, Debug)]
 pub(crate) struct WriteContext<V> {
     version: PhantomData<V>,
+    /// Indicates whether ranges may be "minimized" into single values.
+    /// E.g.: writing `range::[1, 1]` as `1`.
+    pub(crate) minimize_ranges: bool,
```

Comment:
🗺️ The field is `pub(crate)` since this struct isn't supposed to have any special logic to it.